### ISSUE TITLE
Allow comments in Eco grammars

### ISSUE THREAD

**Issue by ptersilie** [softdevteam/eco#85]
Grammars written in Eco don't currently allow comments. Comments would be handy tho.

**Comment by ltratt** [comment]
Agreed. We should use Yacc's comment syntax (`/* ... */`) I guess? Even though it's ugly, it's portable... [Though, personally, I wouldn't object to adding `//` as a secondary comment style, or even instead of `/* ... */`.]

**Comment by ptersilie** [comment]
I'll probably just do both.

**Comment by ltratt** [comment]
I won't object :)
